src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -727,43 +684,5 @@ private static void Sort(List<PackageManagerSearchElementViewModel> results, Pac
                     break;
             }
         }
-
-        /// <summary>
-        ///     A KeyHandler method used by SearchView, increments decrements and executes based on input.
-        /// </summary>
-        /// <param name="sender">Originating object for the KeyHandler </param>
-        /// <param name="e">Parameters describing the key push</param>
-        public void KeyHandler(object sender, KeyEventArgs e)
```

Comment:
Seems like it's key navigation during search, but I didn't find any place, where it's used. Removed it.